mirai-core-mock/src/utils/image.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2019-2022 Mamoe Technologies and contributors.
+ *
+ * 此源代码的使用受 GNU AFFERO GENERAL PUBLIC LICENSE version 3 许可证的约束, 可以在以下链接找到该许可证.
+ * Use of this source code is governed by the GNU AGPLv3 license that can be found through the following link.
+ *
+ * https://github.com/mamoe/mirai/blob/dev/LICENSE
+ */
+
+package net.mamoe.mirai.mock.utils
+
+import net.mamoe.mirai.utils.randomImageContent as miraiutils_randomImageContent
+
+// Make `randomImageContent` public
+public fun randomImageContent(): ByteArray = miraiutils_randomImageContent()
```

Comment:
```suggestion
// Make `randomImageContent` public
public fun Image.Companion.randomImageContent(): ByteArray = miraiutils_randomImageContent()
```
减少top-level functions